src/components/ItaliaTheme/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -25,17 +29,72 @@ const messages = defineMessages({
 const Breadcrumbs = ({ pathname }) => {
   const intl = useIntl();
   const dispatch = useDispatch();
+  const location = useLocation();
 
-  let items = useSelector((state) => state.breadcrumbs.items, isEqual);
+  let items =
+    useSelector((state) => {
+      return state.breadcrumbs.items;
+    }, isEqual) || [];
   const subsite = useSelector((state) => state.subsite?.data);
+  const bcLoaded = useSelector((state) => {
+    return state.breadcrumbs.loaded;
+  });
+
+  // Funzione per fare match di routes
+  const getMatchingRoute = (p) =>
+    matchPath(location.pathname, p) !== null ||
+    matchPath(location.pathname, p.replace('**/', '')) !== null;
+
+  // Funzione per riconoscere se siamo in una route statica
+  const getCurrentPathFromAddonRoutes = () =>
+    config.addonRoutes.find((route) => {
+      const paths = typeof route.path === 'string' ? [route.path] : route.path;
+      return paths.find(getMatchingRoute);
+    }) || {};
+
+  // Variabile per riconoscere se esistono in config route statiche con il path corrente
+  const pathIsCustomAddonRoute = config.addonRoutes.some((route) => {
+    const paths = typeof route.path === 'string' ? [route.path] : route.path;
+
+    return (
+      paths.filter(getMatchingRoute).length > 0 &&
+      route.breadcrumbs_title !== null
+    );
+  });
```

Comment:
Forse non serve? Basta controllare se sta nell'array `config.settings.nonContentRoutes`?

Reply:
mmm no in realtà li fa di più.. verifica se è stato definito un breadcrumb title

Reply:
Vero, però vedo che lo verifica poi anche sotto prima di aggiungerlo

Reply:
su questa parte ci aveva lavorato @deodorhunter per ultima. chiederei meglio a lei il perchè di questo. Guardando bene tutto il codice penso che hai ragione tu @pnicolli, probabilmente pathIsCustomAddonRoute è superfluo e forse la if alla riga 66 si può togliere lasciando solo quello che c'è scritto al suo interno

Reply:
Ho aggiornato questa parte togliendo il check ridondante a monte, testato mi sembra funzionante per le casistiche base di io-comune (/search) e per quelle custom che abbiamo visto su reggioemilia